style/README.md

```diff
@@ -1,32 +1,37 @@
-Style
-=====
+# Style
 
-A guide for programming in style.
+At Healthify, we determine code conventions by team consensus. Our aim is to create an environment that is conducive to a high level of code quality and that developers enjoy coding in.
 
-Use [Hound] to automatically review your
-GitHub pull requests for
-style guide violations.
+We enforce code styles with [Hound](hound) during CI. You can also use [RuboCop](rubocop) locally to enforce styles for Ruby code. To see our most up to date styles for the languages we use, check out the following files in our [main repository](main-repo) (private access):
```

Comment:
It looks like we also use `jshint` locally (and via Hound?). Is it worth mentioning that here too?

Reply:
Will do, @joofsh !